starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -27,12 +78,48 @@ async def syncing() -> Union[dict, bool]:
     return False
 
 
-# pylint: disable=redefined-builtin
-async def get_events(filter: dict) -> dict:
+# pylint: disable=too-many-arguments
+async def get_events(
+    from_block: BlockId,
+    to_block: BlockId,
+    address: Address = "",
```

Comment:
If `""` bears some special meaning, it should be extracted, and the same constant should be used in all places in the code.

Reply:
Is not special, it's just an empty value like 0 in ints.